### DISCUSSION TITLE
Stepper changes Temperament without having been asked to do so in making the .cmb file

### DISCUSSION THREAD

**Discussion by avdeursen** [GrandOrgue/grandorgue#1057] [Help]
Last week I upgraded Grandorgue from 3.1 installed a few years ago to 3.6. Existing cmb files have different lengths, .txt files are about 1 Mb, compacter binary files are 36 kB. The stops come up very well in the 3.6 version. But remarkably enough, the Temperament sometimes changes from original to ¼ tone upon stepping. And this also occurs after introducing the stepper sequence from scratch. Is this a known issue, of perhaps a bug? I hope that somebody in the Grandorgue community can advise.

Attached is the most recent small .cmb file for Fantasie en La by C. Franck on the Friesach organ by Piotr Grabowski where this occurs going to step 3. It is packed in a zip file.
[Franck Fantasie en La.zip](https://github.com/GrandOrgue/grandorgue/files/8156121/Franck.Fantasie.en.La.zip)

**Comment by willeke1234** [comment]
In the file, SetterNext/SetterPrev and SetterTemperamentNext/SetterTemperamentPrev have the same MIDI event setting. Is the temperament is stepping along because it reacts to the sequencer MIDI events?

**Reply by avdeursen** [reply]
Thanks Willeke, It should be something like that, but I have no idea how that setting got there. To solve the problem, I reinstalled Friesach, after I checked that the initial midi settings did not contain such double midi event setting. It now looks like the stepper is working properly. And I will double check all my other .cmb files.
